style-rtl.css

```diff
@@ -1,3 +1,4 @@
+@charset "UTF-8";
```

Comment:
@EvanHerman I think this branch might be behind `develop` a little bit. I introduced a change that prevents these from happening because WordPress.org theme reviewers asked us to remove them.

You may need to merge in the latest from `develop` then run `npm install && grunt`.

cf. https://github.com/godaddy/wp-primer-theme/blob/5b2e377bb308b53166bafd5cd769b8118e8c7dd1/Gruntfile.js#L186-L195